tests/tests/swfs/avm2/click_invisible/Test.as

```diff
@@ -0,0 +1,46 @@
+﻿package {
```

Comment:
(Nit: I do not know if there exists any auto-formatter for ActionScript, so I manually indented this file:

https://gist.github.com/iwannabethedev/00e73648a53f137873c2d8f229f6d30a

I did it quickly just by `Shift+Tab`'ing the whole selection a couple of times in a text editor like Gedit, and then `Tab` selections a few times as appropriate.)